source/MaterialXContrib/MaterialXNode/resources/tiledImage.xml

```diff
@@ -21,7 +21,7 @@
     <int name="tiled_image3_frameendaction" value="1" />
   </values>
   <outputs>
-    <float3 name="out" />
+    <float3 name="tiled_image3_output" />
```

Comment:
Now that we use the correct output name from the shader graph we need to make sure that we don't use the previous hard-coded "out"name but tiled_image3_output (which matches the mltx file). Both the texture + shader graph now render properly.